src/main/charts/bitbucket/templates/statefulset.yaml

```diff
@@ -136,7 +136,8 @@ spec:
           volumeMounts:
             - name: local-home
               mountPath: {{ .Values.volumes.localHome.mountPath | quote }}
-            {{- if or .Values.volumes.sharedHome.persistentVolumeClaim.create .Values.volumes.sharedHome.customVolume }}
+            {{- if or .Values.volumes.sharedHome.persistentVolumeClaim.create .Values.volumes.sharedHome.customVolume
```

Comment:
I think it will be useful to have a comment here explaining why we have this condition. Something like

{{/* jmxExporter is saved to shared home but mirrors don't have one, so we need at least emptyDir */}}